### ISSUE TITLE
Append parameter names in method for simple types

### ISSUE THREAD

**Issue by dgalichet** [scalameta/metals-feature-requests#229]
### Is your feature request related to a problem? Please describe.

This feature is related to https://github.com/scalameta/metals-feature-requests/issues/184 that proposes to show parameter names for simple types

It could be nice to have a code action to append these parameters directly on the source code (according to @tgodzik ) it should be pretty simple to do)

### Describe the solution you'd like

```scala
def easyMethod(boolParam: Boolean) = ???
easyMethod(true)
```

After invoking code action, the code should be:
```scala
easyMethod(boolParam = true)
```

### Describe alternatives you've considered

the alternative is to show parameter only, without inserting it as described in https://github.com/scalameta/metals-feature-requests/issues/184

### Search terms

named parameters decorations inferred information

**Comment by benjamingeer** [comment]
It would be great to have a refactor that would add names to all arguments in a method call, regardless of their types. Something like IntelliJ's "use named arguments for current and subsequent arguments":

![image](https://user-images.githubusercontent.com/558389/153206840-bd4dace5-21bc-41ab-9072-ae19e9ccf8f6.png)

**Comment by tanishiking** [comment]
It would be nice to use [Inlay Hint](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_inlayHint) for https://github.com/scalameta/metals-feature-requests/issues/184

Inlay Hint can have `textEdits` field, and we would be able to click the hints to insert parameter names.

```
/**
 * Optional text edits that are performed when accepting this inlay hint.
 *
 * *Note* that edits are expected to change the document so that the inlay
 * hint (or its nearest variant) is now part of the document and the inlay
 * hint itself is now obsolete.
 *
 * Depending on the client capability `inlayHint.resolveSupport` clients
 * might resolve this property late using the resolve request.
 */
textEdits?: TextEdit[];
```

**Comment by tanishiking** [comment]
Let me re-open this issue as it's not yet available for Scala3.
Scala2 support will be available with Metals 0.11.7, thank you very much for working on this @camgraff :tada: !
